frontend/src/app/testResults/uploads/UploadForm.tsx

```
@@ -348,21 +348,25 @@ const UploadForm: React.FC<UploadFormProps> = ({
         });
       } else {
         const response = await res.json();
-        // failed upload due to validation errors
-        if (response?.length === 1 && response[0].errors?.length) {
+        const allErrors = response.flatMap(
+          (item: { errors: any[] }) => item.errors
+        );
+
+        // failed upload due to validation errors from either Simple Report or Report Stream
+        if (allErrors.length > 0) {
```

It looks like we were assuming that any validation errors strictly came from our own validation prior to uploading to Report Stream Covid/Universal pipelines and did not handle validation errors coming from Report Stream. Report Stream validation errors were resulting in a silent failure.